workshops/base_views.py

```diff
@@ -206,3 +206,19 @@ def get_success_url(self):
             return next_url
         else:
             return default_url
+
+
+class PrepopulationSupportMixin:
+
+    def get_initial(self):
+        return self.request.GET.dict()
```

Comment:
> Can you show an use case where we would not like to have a field prepopulated even though it exists as a GET parameter?

I cannot right now, but I'm worrying about security issues - like if it's possible to override value for `id` field (seems not possible). We may have name collisions, i.e. when we define `next` field on a model and, at the same time, we support redirection (`RedirectSupportMixin`). Right now I cannot imagine any scenario where this behavior could be dangerous, so let's stick with your way. We can always add `ALLOW_PREPOPULATE` later.